projects/cds/src/merchandising/connectors/strategy/converters.ts

```diff
@@ -1,23 +1,13 @@
 import { InjectionToken } from '@angular/core';
 import { Breadcrumb, Converter } from '@spartacus/core';
 import { MerchandisingFacet } from '../../model/merchandising-facet.model';
-import {
```

Comment:
same as below, I think we can remove MerchandisingProducts from the model because you switched it to StrategyProducts

Reply:
Yup, the models have gone